Evaluation/Input/evaluation_plan.json

```diff
@@ -328,85 +311,82 @@
             "Project": "Alfentanil",
             "Name": "Alfentanil",
             "Type": "Compound",
-            "SectionId": 31
+            "SectionReference": "final-input-parameters"
         },
         {
             "Project": "Alfentanil",
             "Name": "Solution",
             "Type": "Formulation",
-            "SectionId": 31
+            "SectionReference": "final-input-parameters"
         }
     ],
     "Sections": [
         {
-            "Id": 1,
-            "Title": "1 Introduction",
+            "Reference": "introduction",
+            "Title": "Introduction",
             "Content": "Content/Section1_Introduction.md"
         },
         {
-            "Id": 2,
-            "Title": "2 Methods",
-            "Content": "Content/Section2_Methods.md",
+            "Reference": "methods",
+            "Title": "Methods",
```

Comment:
no need to refer **empty** content files anymore. IF the chapter has only subchapters but no content - just remove the "Content" field.